bin/tkldev-makefile-info

```diff
@@ -66,8 +68,10 @@ broader applications)
                 return '\n'.join(out)
             else:
                 return out
-
-    data = mkparser.parse_makefile(args.makefile).to_dict()
+    try:
+        data = mkparser.parse_makefile(args.makefile).to_dict()
+    except FileNotFoundError as e:
+        fatal("Makefile not found - please ensure that you are in app build code base dir")
```

Comment:
We should probably `abspath` the argument instead. Or somehow leverage the locator code in tkldev-detective so we can make the `makefile` argument optional.

Reply:
this is what you meant right?: https://github.com/turnkeylinux/tkldev-detective/pull/4/commits/847bd18d09ddcd3034e56586081653dd5eb06bb8